package.json

```diff
@@ -9,7 +9,7 @@
     "co": "sui-studio commit",
     "commitmsg": "validate-commit-msg",
     "deploy": "sui-deploy spa sui-components --now -p",
-    "generate": "sui-studio generate -S schibstedspain",
```

Comment:
The convention of "@s-ui" was decided only for compliant components. I would not merge this yet.